meson_options.txt

```diff
@@ -0,0 +1 @@
+option('build-pty', type : 'boolean', value : true)
```

Comment:
Cool. I see from the Meson docs we can also use `get_option()` to retrieve info like the installation prefix which might prove useful in the future.

Reply:
Also, this option name is fine for now but as other, optional, binaries are added to the build (e.g., `suid_exec`) it's not obvious if we'll want a config option for each program or options that manage groups of programs.